BUILD.md

```diff
@@ -0,0 +1,65 @@
+# Building the project
+
+We use [Gradle build flavors](http://developer.android.com/tools/building/configuring-gradle.html#workBuildVariants) to enable a number of different build variants of MUSER.
+
+For now, We have one  Gradle "platform" flavor dimensions:
+
+* **Google** = Normal Google Play release
+
+...and one Gradle "brand" flavor dimensions:
```

Comment:
See above - there is no `brand` flavor dimension. I would just copy/paste the text from my above comment describing how the variants are generated, list all possible combined ones, and then just explain which matter to MUSER.

Reply:
Also, should I use the word "Shuttle" or should I say "MUSER"?

Reply:
I would say that this app is derived from Shuttle, which has these variant combinations. But, in the case of MUSER, these are the variants that we're using.